indexing-service/src/main/java/io/druid/indexing/common/task/TaskUtils.java

```diff
@@ -26,8 +26,7 @@
 {
   public static String makeId(String id, final String typeName, String dataSource, Interval interval)
```

Comment:
Hmmm, In that case, I guess we can move this method to AbstractTask itself and remove this class.